test/Microsoft.Azure.WebJobs.Host.UnitTests/Executors/DefaultStorageCredentialsValidatorTests.cs

```diff
@@ -63,7 +58,7 @@ public void StorageAccount_QueueCheckThrows_BlobOnly()
                 .Verifiable();
 
             blobClientMock.Setup(b => b.GetServicePropertiesAsync(It.IsAny<CancellationToken>()))
-                .ReturnsAsync(null);
+                .ReturnsAsync((ServiceProperties)null);
```

Comment:
It looks like Moq added some new overloads to ReturnsAsync so it gets confused if you just return null now. Explicitly casting null to a specific object fixes it. There's a few of these sprinkled throughout this change.